internal/transform/schema.go

```diff
@@ -293,3 +293,146 @@ type SponsorshipOutput struct {
 	Operation      xdr.Operation
 	OperationIndex uint32
 }
+
+// // EffectOutput is a representation of an operation that aligns with the BigQuery table history_effects
```

Comment:
nit: extra comment line 
```suggestion
// EffectOutput is a representation of an operation that aligns with the BigQuery table history_effects
```